cloudfront.tf

```diff
@@ -62,6 +62,22 @@ resource "aws_cloudfront_distribution" "website" {
     prefix          = "${var.hostname}/cloudfront"
   }
 
+  price_class = "PriceClass_100"
+
+  custom_error_response {
```

Comment:
I'm going to update the `custom_error_response` lines here too. I'm uncomfortable with them being unconfigurable as firstly they also don't point to `${var.error_document}` (which is used in the S3 configuration for the content bucket) and you should not change the response code - `404` and `400` are valid responses; browsers and client devices should be sent them to deal with appropriately.

Reply:
sorry about this, my first commit was just to fix what I propose in pull request but i host a angular application and I will move to another branch. what I propose is only the force destroy directive.

Reply:
That's OK. You'll need to either delete the last few commits on this branch and force push it back up to remove them from the pull request or revert them and I'll squash them on the merge.

Reply:
done sorry about the late